src/PrestaShopBundle/Resources/config/routing/admin/improve/international/tax_rules_groups.yml

```
@@ -0,0 +1,82 @@
+admin_tax_rules_groups_index:
+  path: /
+  methods: [GET]
+  defaults:
+    _controller: PrestaShopBundle:Admin\Improve\International\TaxRulesGroup:index
+    _legacy_controller: AdminTaxRulesGroup
+#    _legacy_link: AdminTaxRulesGroup
+
+admin_tax_rules_groups_create:
+  path: /new
+  methods: [GET, POST]
+  defaults:
+    _controller: PrestaShopBundle:Admin\Improve\International\TaxRulesGroup:create
+    _legacy_controller: AdminTaxRulesGroup
+#    _legacy_link: AdminTaxRulesGroup:addtax_rules_group
+
+admin_tax_rules_groups_edit:
+  path: /{taxRulesGroupId}/edit
+  methods: [GET, POST]
+  defaults:
+    _controller: PrestaShopBundle:Admin\Improve\International\TaxRulesGroup:edit
+    _legacy_controller: AdminTaxRulesGroup
+#    _legacy_link: AdminTaxRulesGroup:updatetax_rules_group
+    _legacy_parameters:
+      id_tax_rules_group: taxRulesGroupId
+  requirements:
+    taxRulesGroupId: \d+
+
+admin_tax_rules_create:
+  path: /{taxRulesGroupId}/tax-rules/new
+  methods: [POST]
+  defaults:
+    _controller: PrestaShopBundle:Admin\Improve\International\TaxRulesGroup:createTaxRule
+    _legacy_controller: AdminTaxRulesGroup
+#    _legacy_link: AdminTaxRulesGroup:addtax_rules_group
+  requirements:
+    taxRulesGroupId: \d+
+
+admin_tax_rules_load:
+  path: /tax-rules/{taxRuleId}/edit
```

```suggestion
  path: /tax-rules/{taxRuleId}/load
```
Dont use the same URL pattern for `admin_tax_rules_load` and `admin_tax_rules_edit` it can lead to huge issues 😉

---

This was done on purpose so that i wouldn't have to extend extend grid rendering to add data-attribute with path to this new route or without creating separate action for this case.

---

Nice try ... but I prefer we create separate actions 😄 . More code, but less magic and more explicit => easier to debug.

I prefer more code, easier to maintain and to debug, than less code easier to modify 😉